Patch added by Enrique Robledo Arnuncio <era@debian.org> in version 0.5.3-1.

--- ../orig/freqtweak-0.7.0~cvs20070605.orig/freqtweak.1	2003-08-20 05:22:36.000000000 +0200
+++ freqtweak.1	2007-06-05 10:23:41.000000000 +0200
@@ -61,6 +61,10 @@
 high-resolution spectral displays in the form of scrolling-raster
 spectrograms and energy vs frequency plots displaying both pre and
 post-processed spectra.
+
+Freqtweak is a jack client. You need to have jackd running in order to
+get sound input and output with freqtweak.
+
 .SH MOUSE CONTROL
 .P
 The processing filters are controlled through mouse operations on each
